backends/tfhe-cuda-backend/cuda/include/integer.h

```diff
@@ -1035,6 +1035,7 @@ template <typename Torus> struct int_fullprop_buffer {
                uint32_t gpu_count) {
 
     lut->release(streams, gpu_indexes, 1);
+    delete lut;
```

Comment:
question, I saw this pop up in the updates, gpu_count is ignored and 1 is hardcoded instead ?

Reply:
This is for the full propagation where the LUT is applied to only one input, so I hardcoded the value to 1 here yes.